deploy/examples/crds.yaml

```diff
@@ -5797,7 +5799,6 @@ spec:
                   type: object
                   x-kubernetes-preserve-unknown-fields: true
               required:
-                - dataPools
```

Comment:
Should `dataPools` really be optional now?

Reply:
keeping required makes sense

Reply:
Agreed, it still stays required